File: server/global/var.go
package global

import (
	"flag"

	"github.com/casbin/casbin/v2"
	"github.com/codepzj/stellux/server/internal/pkg/apiwrap"
	"github.com/gin-gonic/gin"
)

// 全局变量
var (
	Enforcer *casbin.Enforcer
)

// 命令行参数
var (
	Mode = flag.String("mode", "development", "运行模式,eg: development/production")
)

// 常量
var (
	AccessTokenNotFound  = gin.H{"code": apiwrap.RequestAccessTokenNotFound, "msg": "未携带access_token"}
	AccessTokenExpired   = gin.H{"code": apiwrap.RequestAccessTokenExpired, "msg": "access_token已过期"}
	LoadPermissionFailed = gin.H{"code": apiwrap.RequestLoadPermissionFailed, "msg": "权限加载失败"}
	PermissionDenied     = gin.H{"code": apiwrap.RequestPermissionDenied, "msg": "权限不足,禁止访问"}
)

File: admin/src/global/modules/var.ts
export const Code = {
  RequestSuccess: 0, // 请求成功
  RequestBadRequest: 1, // 请求参数错误
  RequestUnauthorized: 2, // 未授权
  RequestForbidden: 3, // 禁止访问
  RequestNotFound: 4, // 未找到
  RequestInternalServerError: 5, // 服务器错误
  RequestAccessTokenExpired: 6, // access_token已过期
  RequestAccessTokenNotFound: 7, // access_token未找到
  RequestRefreshTokenExpired: 8, // refresh_token已过期
  RequestRefreshTokenNotFound: 9, // refresh_token未找到
  RequestPermissionDenied: 10, // 权限不足
  RequestLoadPermissionFailed: 11, // 权限加载失败
};
